Exporter/src/main/java/uk/ac/ebi/biostd/exporter/service/SubmissionService.java

```diff
@@ -57,52 +60,60 @@ public Submission getSubmission(String accNo) {
         return submission;
     }
 
-    private List<String> getAccessTags(Submission submission) {
+    private List<String> getAccessTags(Submission submission, List<String> accessTags) {
         List<String> tags = new ArrayList<>();
         tags.add(submissionDao.getUserEmail(submission.getOwner_id()));
-        tags.addAll(submissionDao.getAccessTags(submission.getId()));
+        tags.addAll(accessTags);
         tags.add("#" + submission.getOwner_id());
-
         return tags;
     }
 
-    private List<Attribute> getAttributes(Submission submission) {
+    private List<Attribute> getAttributes(Submission submission, List<String> accessTags) {
         List<Attribute> subAttributes = submissionDao.getAttributes(submission.getId());
-        subAttributes
-                .add(new Attribute("ReleaseDate", DateUtils.getFromEpochSeconds(Long.valueOf(submission.getRTime()))));
+        subAttributes.add(new Attribute("ReleaseDate", getFromEpochSeconds(parseLong(submission.getRTime()))));
         subAttributes.add(new Attribute("Title", submission.getTitle()));
+        accessTags.forEach(tag -> addProjectAttribute(subAttributes, tag));
         return subAttributes;
     }
 
+    private void addProjectAttribute(List<Attribute> subAttributes, String accessTag) {
+        if (accessTag.equals("Public")) {
+            return;
+        }
+
+        if (subAttributes.stream().noneMatch(attribute -> attribute.getName().equals(accessTag))) {
```

Comment:
I don't get this logic, why are you trying to find an attribute named like a tag? Also, wouldn't this add other tags such as user id as `AttachTo` attribute? :thinking:

Reply:
1. Some submission has attach to attribute and access tag. New submitter remove duplication and now we only store attach to so basically here I check if attribute is there (is an old submission) I do not added , if attribute is there (is a new submission) I added it. 

2. About user id and all that, this is not stored in db, this is added artificially that is the reason that I check only stored tags and not all generated access tag create in this class.

Reply:
Got it